privacyidea/lib/policy.py

```diff
@@ -2516,3 +2545,49 @@ def generic(cls, g, scope=None, realm=None, resolver=None, user=None, user_objec
                    client=client, action=action, adminrealm=adminrealm,
                    adminuser=adminuser, time=time,
                    sort_by_priority=sort_by_priority)
+
+
+def check_pin(g, pin, tokentype, user_obj):
```

Comment:
I actually gave it some thoughts. There was another function, that is called check_pin_policy.
As this function is contained in the module ``policy`` I decided to only name it ``check_pin``, since it seems obvious, that a check_pin function in a module "policy" would acutally check pins for policies ;-)